app/src/main/java/nl/tudelft/pixelperfect/client/ClientListener.java

```diff
@@ -17,31 +18,31 @@
  * @author Floris Doolaard
  * @author Dmitry Malarev
  */
+@SuppressWarnings("unused")
 class ClientListener implements MessageListener<Client> {
     public void messageReceived(Client source, Message message) {
-        if (message instanceof HelloMessage) {
-            // do something with the message
-            HelloMessage helloMessage = (HelloMessage) message;
-            System.out.println("Client #"+source.getId()+" received: '"+helloMessage.getSomething()+"'");
-        } else if (message instanceof EventsMessage) {
+        if (message instanceof EventsMessage) {
             Event mission;
             EventsMessage eve = (EventsMessage) message;
             System.out.println("Client #"+source.getId()+" received event: '"+eve.getType()+"'");
             switch (eve.getType()) {
                 case "FireEvent":
                     mission = new FireEvent(eve.getID(), "", "", eve.getTime(), eve.getDuration(), 0);
```

Comment:
maybe we should refactor this too, however this will involve changes in the Client, Server and messages, so this might be a project for next week (or perhaps tomorow morning)